File: https-quera.org-problemset-33045/Solution.py
from math import sqrt

def app():
    num = int(input())
    divisor = 0
    sm = 0
    for n in range(1, num + 1):
        divisors = list(divisorGenerator(n))
        divisor += len(divisors)
        sm += sum(divisors)

    print(divisor, int(sm))



def divisorGenerator(num):
    large_divisors = []
    for i in range(1, int(sqrt(num) + 1)):
        if num % i == 0:
            yield i
            if i*i != num:
                large_divisors.append(num / i)
    for divisor in reversed(large_divisors):
        yield divisor

if __name__ == "__main__":
    app()

File: https-quera.org-problemset-33045/Solution.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }

    public static void app() {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        int divisorCount = 0;
        long sum = 0;

        for (int n = 1; n <= num; n++) {
            List<Integer> divisors = divisorGenerator(n);
            divisorCount += divisors.size();
            for (int div : divisors) {
                sum += div;
            }
        }

        System.out.println(divisorCount + " " + sum);
    }

    public static List<Integer> divisorGenerator(int num) {
        List<Integer> divisors = new ArrayList<>();
        List<Integer> largeDivisors = new ArrayList<>();

        for (int i = 1; i <= Math.sqrt(num); i++) {
            if (num % i == 0) {
                divisors.add(i);
                if (i * i != num) {
                    largeDivisors.add(num / i);
                }
            }
        }
        for (int i = largeDivisors.size() - 1; i >= 0; i--) {
            divisors.add(largeDivisors.get(i));
        }
        return divisors;
    }
}
